<!-- Kaqi plane slam -->
<launch>

  <node pkg="plane_slam" type="plane_slam_node" name="plane_slam_node" output="screen" >
    <param name="subscriber_queue_size" value="4"/>
    <param name="topic_image_visual" value="/head_kinect/rgb/image_rect_color"/>
    <param name="topic_image_depth" value="/head_kinect/depth_registered/image"/>
    <param name="topic_camera_info" value="/head_kinect/depth_registered/camera_info"/>
    <param name="topic_point_cloud" value=""/>
  </node>

</launch>
